communitybsl.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Code Analyzer for 1C (BSL)
 homepageUrl=https://1c-syntax.github.io/sonar-bsl-plugin-community/en
-archivedVersions=1.11.0,1.12.0
-publicVersions=1.13.0
+archivedVersions=1.11.0,1.12.0,1.13.0
+publicVersions=1.14.0
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-communitybsl-plugin
 
+1.14.0.description=Localization, optimization, 4 new diagnostics, fixes
+1.14.0.sqVersions=[9.9,LATEST]
+1.14.0.date=2024-06-05
+1.14.0.changelogUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/tag/v1.14.0
+1.14.0.downloadUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/download/v1.14.0/sonar-communitybsl-plugin-1.14.0.jar
+
 1.13.0.description=Speed optimization
-1.13.0.sqVersions=[9.9,LATEST]
+1.13.0.sqVersions=[9.9,9.9.*]
```

Comment:
Ending this range at 9.9.* means your current users on 10.0 - 10.5.1 won't be prompted to upgrade. 
Is this about the `requiredForLanguages` stuff? That starts in 10.4 but is off by default. On by default in 10.5

Reply:
The range has been corrected.

I don’t understand the second question, can you please explain?